features/admin.actions.v1/api/create-action.ts

```diff
@@ -39,7 +39,8 @@ const httpClient: HttpClientInstance = AsgardeoSPAClient.getInstance()
  * @returns Promise containing the response.
  * @throws Throws an IdentityAppsApiException if the request fails.
  */
-const createAction = (actionType: string, actionBody: ActionInterface): Promise<ActionResponseInterface> => {
+const createAction = (actionType: string, actionBody: ActionInterface |
+    PreUpdatePasswordActionInterface): Promise<ActionResponseInterface> => {
```

Comment:
Let's use generics to accept any interface that extends ActionInterface for the actionBody.

```suggestion
const createAction = <T extends ActionInterface>(actionType: string, actionBody: T): Promise<ActionResponseInterface> => {
```

Reply:
Updated in

Reply:
> Updated in

The commit hash seems to be omitted from the reply